src/app/beer_garden/scheduler.py

```diff
@@ -490,8 +498,8 @@ def run_job(job_id, request_template, **kwargs):
         request = beer_garden.router.route(
             Operation(
                 operation_type="REQUEST_CREATE",
-                model=request_template,
-                model_type="RequestTemplate",
+                model=Request.from_template(request_template),
```

Comment:
This change is already on develop, which made me look at the git log on this branch.  It looks like your starting point is pretty far off from the current develop.  Try to clean that up and make the current head of develop your starting point so the the PR only shows things you're actually changing here.